core/api/tests/test_business_plans.py

```diff
@@ -121,139 +116,118 @@ def test_list_year_filter(self, user, _setup_bp_list):
+    status = "Endorsed"
+    year_start = 2025
+    year_end = 2027
+    params = f"?status={status}&year_start={year_start}&year_end={year_end}"
+    url = reverse("bp-upload") + params
+
+    def test_without_login(self):
+        response = self.client.post(self.url, {}, format="multipart")
+        assert response.status_code == 403
 
-    def test_bp_create(
+    def test_bp_import(
         self,
-        agency_user,
+        user,
+        meeting,
+        decision,
         agency,
         country_ro,
-        substance,
-        sector,
-        subsector,
         project_type,
         bp_chemical_type,
+        substance,
+        sector,
+        subsector_other,
         _setup_bp_activity_create,
-        _setup_new_business_plan_create,
         mock_send_mail_bp_create,
     ):
-        self.client.force_authenticate(user=agency_user)
-
-        data = _setup_new_business_plan_create
-        activity_data = _setup_bp_activity_create
-        data["activities"] = [activity_data]
+        self.client.force_authenticate(user=user)
+        url = f"{self.url}&meeting_id={meeting.id}&decision_id={decision.id}"
 
-        response = self.client.post(self.url, data, format="json")
-        assert response.status_code == 201
-        assert response.data["name"] == "Test BP"
-        assert response.data["status"] == "Endorsed"
-        assert response.data["year_start"] == 2020
-        assert response.data["year_end"] == 2023
+        with open(self.file_path, "rb") as f:
+            data = {"Test_BP2025-2027.xlsx": f}
+            response = self.client.post(url, data, format="multipart")
 
-        activities = response.data["activities"]
-        assert activities[0]["business_plan_id"] == response.data["id"]
-        assert activities[0]["title"] == "Planu"
-        assert activities[0]["agency_id"] == agency.id
-        assert activities[0]["country_id"] == country_ro.id
-        assert activities[0]["lvc_status"] == "LVC"
-        assert activities[0]["project_type_id"] == project_type.id
-        assert activities[0]["bp_chemical_type_id"] == bp_chemical_type.id
-        assert activities[0]["substances"] == [substance.id]
-        assert activities[0]["sector_id"] == sector.id
-        assert activities[0]["subsector_id"] == subsector.id
-        assert activities[0]["status"] == "A"
-        assert activities[0]["is_multi_year"] is False
-        assert activities[0]["remarks"] == "Merge bine, bine, bine ca aeroplanu"
-        assert activities[0]["values"][0]["year"] == 2020
+        assert response.status_code == 200
+        assert response.data == "Data imported successfully"
+
+        business_plan = BusinessPlan.objects.get(
+            status=self.status,
+            year_start=self.year_start,
+            year_end=self.year_end,
+            meeting=meeting,
+            decision=decision,
+        )
+        activity = business_plan.activities.first()
```

Comment:
check if the value for Amount of Polyol is correctly set (there is a warning in the validation so we have to make sure we set it correctly)